src/Riak/Api/Http.php

```diff
@@ -166,6 +167,9 @@ protected function buildPath()
             case 'Basho\Riak\Command\MapReduce\Fetch':
                 $this->path = sprintf('/%s', $this->config['mapred_prefix']);
                 break;
+            case 'Basho\Riak\Command\Indexes\Query':
+                $this->path = $this->createIndexQueryPath($this->command, $bucket);
```

Comment:
I did that to type hint it into that type specifically within the `createIndexQueryPath` function.